src/lib.rs

```diff
@@ -37,7 +37,7 @@ impl fmt::Display for Target {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
             Target::Solana => write!(f, "solana"),
-            Target::Substrate { .. } => write!(f, "substrate"),
+            Target::Polkadot { .. } => write!(f, "polkadot"),
```

Comment:
```suggestion
            Target::Polkadot { .. } => write!(f, "Polkadot"),
```

Reply:
I'll do that as a follow up for both targets as this breaks the `contract` tests